Microsoft.Toolkit.Uwp.UI/Converters/DoubleToObjectConverter.cs

```diff
@@ -0,0 +1,160 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+
+namespace Microsoft.Toolkit.Uwp.UI.Converters
+{
+    /// <summary>
+    /// This class converts a double value into an other object.
+    /// Can be used to convert doubles to visibility, a couple of colors, couple of images, etc.
+    /// If GreaterThan and LessThan are both set, the logic looks for a value inbetween the two values.
+    /// Otherwise the logic looks for the value to be GreaterThan or LessThan the specified value.
+    /// The ConverterParameter can be used to invert the logic.
+    /// </summary>
+    [Bindable]
+    public class DoubleToObjectConverter : DependencyObject, IValueConverter
```

Comment:
Need to add to docs

Reply:
I'll create a separate docs PR in the docs repro for the docs for TabView, DoubleToObjectConverter, and FindAncestor.